src/pages-and-resources/live/data/api.js

```diff
@@ -0,0 +1,165 @@
+  }
+
+  return {
+    consumerKey: data.lti_1p1_client_key,
+    consumerSecret: data.lti_1p1_client_secret,
+    launchUrl: data.lti_1p1_launch_url,
+  };
+}
+
+function normalizePiiSharing(data) {
+  return {
+    piiSharing: data.pii_sharing_allowed,
+    piiShareUsername: data.pii_sharing_allowed,
+    piiShareEmail: data.pii_sharing_allowed,
+  };
+}
+
+function normalizeAppConfig(data) {
+  return {
+    id: data.provider_type,
+    ...normalizeLtiConfig(data.lti_configuration),
+  };
+}
+
+function normalizeProviders(data) {
+  const apps = Object.entries(data.available).map(([key, app]) => ({
+    id: key,
+    messages: app.messages,
+    externalLinks: {
+      learnMore: app.external_links.learn_more,
+      configuration: app.external_links.configuration,
+      general: app.external_links.general,
+      accessibility: app.external_links.accessibility,
+      contactEmail: app.external_links.contact_email,
+    },
+    adminOnlyConfig: !!app.admin_only_config,
+  }));
+  return {
+    activeAppId: data.active,
+    apps,
+  };
+}
+
+function normalizeSettings(data) {
+  return {
+    appConfig: normalizeAppConfig(data),
+    piiConfig: normalizePiiSharing(data.lti_configuration),
+  };
+}
+
+function denormalizeData(courseId, appId, data) {
+  const ltiConfiguration = {};
+
+  if (data.consumerKey) {
+    ltiConfiguration.lti_1p1_client_key = data.consumerKey;
+  }
+  if (data.consumerSecret) {
+    ltiConfiguration.lti_1p1_client_secret = data.consumerSecret;
+  }
+  if (data.launchUrl) {
+    ltiConfiguration.lti_1p1_launch_url = data.launchUrl;
+  }
+  if (data.launchEmail) {
+    ltiConfiguration.lti_1p1_launch_email = data.launchEmail;
+  }
+  if ('piiShareUsername' in data) {
+    ltiConfiguration.pii_share_username = data.piiShareUsername;
+  }
+  if ('piiShareEmail' in data) {
+    ltiConfiguration.pii_share_email = data.piiShareEmail;
+  }
+
+  if (Object.keys(ltiConfiguration).length > 0) {
+    // Only add this in if we're sending LTI fields.
+    // TODO: Eventually support LTI v1.3 here.
+    ltiConfiguration.version = 'lti_1p1';
+  }
+
+  const apiData = {
+    context_key: courseId,
+    enabled: true,
+    lti_configuration: ltiConfiguration,
+    provider_type: appId,
+  };
+
+  return apiData;
+}
+
+export function getLiveProvidersUrl() {
+  // return `${getConfig().STUDIO_BASE_URL}/api/live/v0/course/${courseId}/providers`;
+}
+
+export function getLiveSettingsUrl() {
+  // return `${getConfig().STUDIO_BASE_URL}/api/live/v0/course/${courseId}/settings`;
+}
+
+export async function getLiveProviders() {
+  // const { data } = await getAuthenticatedHttpClient()
+  //   .get(getLiveProvidersUrl(courseId));
+
+  const data = {
+    active: 'zoom',
+    available: {
+      zoom: {
+        admin_only_config: true,
+        external_links: {
+          accessibility: '',
+          configuration: '',
+          contact_email: '',
+          general: 'https://edstem.org/us/',
+          learn_more: '',
+        },
```

Comment:
why this duplication of data? why are we not using this object in component and recreating the same object here src/pages-and-resources/live/Settings.jsx

Reply:
Currently. I did not attach the redux with the component. This Dummy data will remove in clean up commit and api integration section will more clear in integration ticket.